samples/echo/func.py

```diff
@@ -42,5 +42,5 @@ def handler(ctx, data: io.BytesIO=None):
 
     content, status, headers = await call
 
-    assert 200 == status
+    assert 202 == status
```

Comment:
200?

Reply:
well, no, there are two layers where the status code changes, with fixtures the status code doesn't change only when FDK transforms an FDK response to actual HTTP response magic happens (see fdk/event_handler.py changes), that's why it's 202, but not 200.